index.html

```diff
@@ -5,8 +5,61 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href = "styles/index.css" rel="stylesheet" type="text/css">
 </head>
 <body>
-    
+    <section id ="headers">
+         <div id ="weather-header"><span class='bigger'>Weather Report</span> &nbsp;&nbsp; 
+            <span class="small">For the lovely city of </span> ✨<span class="dynamic">city</span>✨</div>
+         <!-- <div id ="lovely-city"> For the lovely city of</div>
+         <div id ="header-city">here change the city dynamically</div> -->
+    </section>
+<section id ="whole-body">
```

Comment:
This probably wants to be a `main` tag:

```suggestion
<main>
```